docs/sdk/v3/guides/03-web3-development-basics.md

```diff
@@ -0,0 +1,157 @@
+---
+id: web3-development-basics
+title: Web3 Development Basics
+position: 3
+---
+
+## Introduction
+
+Developing dApps and interacting with Smart Contracts is quite different from Web2, and at times challenging due to little information
+on this topic compared to other areas of software development.
+
+This developer guide is a quick overview of the space, including references to libraries and guides that are great starting points.
+
+It is assumed that you know the basics about Ethereum and the blockchain, including some terminology.
+
+## RPCs
+
+The access point to the blockchain are RPCs. They are the [standardized interface](https://ethereum.org/en/developers/docs/apis/json-rpc/)
+to read data from smart contracts, send transactions and interact with on-chain protocols.
+
+RPCs are basically [full or archival nodes](https://ethereum.org/en/developers/docs/nodes-and-clients/archive-nodes/)
+with an interface ([JSON-RPC](https://www.jsonrpc.org/)).
+
+To support Ethereum's decentralization, once can host a node themselves using one of the implementations listed below:
+
+* [geth](https://github.com/ethereum/go-ethereum) - The original (reference) implementation of the Ethereum protocol
+* [erigon](https://github.com/ledgerwatch/erigon) - A very efficient archival node implementation
+* [Nethermind](https://github.com/NethermindEth/nethermind) - An Ethereum implementation focused on stability
+
+As achieving high availability and making sure your node is synced up all the time turns out to be quite the challenge,
+there are nodes as a service (RPC) providers that you can use, especially in production environments.
```

Comment:
I would add a bit more on what to look for here. For example Infura is not usable for nearly all chains as it only production-supports websockets on Mainnet and beta-supports them on Polygon and Linea(???).
Alchemy has lots of non-standard endpoints which is also awful.
Chainnodes has everything we want but I'd add a bit more general information on what a developer needs from a RPC provider.